gw_eccentricity/gw_eccentricity.py

```diff
@@ -126,23 +130,29 @@ def measure_eccentricity(tref_in=None,
           quasicircular counterpart from the omega22(t) of the eccentric
           waveform.
         - "AmplitudeFits": Uses Amp22(t) and iteratively subtracts a
-          PN-inspired fitting function from it, and finds extrema of the
-          residual.
+          PN-inspired fit of the extrema of Amp22(t) from it, and finds extrema
+          of the residual.
         - "FrequencyFits": Uses omega22(t) and iteratively subtracts a
-          PN-inspired fitting function from it, and finds extrema of the
-          residual.
+          PN-inspired fit of the extrema of omega22(t) from it, and finds
+          extrema of the residual.
         Default is "Amplitude".
-        Available list of methods can be also obtained from
+        The available list of methods can be also obtained from
         gw_eccentricity.get_available_methods().
+        Detail description of these methods can be found in our paper,
```

Comment:
Detail -- > Detailed.
Out of curiosity, is Grammarly smart enough to catch stuff like this?

Reply:
remove "our paper", just say Sec and arxiv number.

Reply:
Done